crates/algod_client/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "algod_client"
+version = "0.0.1"
+authors = ["OpenAPI Generator"]
+description = "API endpoint for algod operations."
```

Comment:
```suggestion
description = "API client for algod interaction."
```

Reply:
this is taken straight from oas spec - that seems to be official description for the algod v2 spec we take from go algorand repo - do we want to modify convert oas script to modify the description when it converts it to v3 spec? https://github.com/algorand/go-algorand/blob/9adaf5573757900f2e57f1f656704176ece8d65a/daemon/algod/api/algod.oas2.json#L7

Reply:
We describing the crate that is being produce, so probably makes sense to adjust the OAS for our generation meta.